TESTS/mbedmicro-rtos-mbed/malloc/main.cpp

```diff
@@ -54,17 +54,22 @@ void task_using_malloc(void)
 
 int main()
 {
+    // static stack for threads to reduce heap usage on devices with small RAM
+    // and eliminate run out of heap memory problem
+    MBED_ALIGN(8) uint8_t stack[THREAD_STACK_SIZE * NUM_THREADS];
```

Comment:
The alignment isn't necessary with #5405.

If you make this a two dimensional array then the loop below will be clearer.

Declaration:
```
uint8_t stack[NUM_THREADS][THREAD_STACK_SIZE];
```
Loop:
```
thread_list[i] = new Thread(osPriorityNormal, THREAD_STACK_SIZE, stack[i]);
```